packages/block-library/src/navigation-menu/index.php

```diff
@@ -56,18 +89,29 @@ function build_navigation_menu_html( $block ) {
  * @uses render_block_navigation_menu()
  */
 function register_block_core_navigation_menu() {
+	$block_content = file_get_contents ( dirname( __FILE__ ) . '/../../../packages/block-library/src/navigation-menu/block.json' );
```

Comment:
Is there a better way to get this path? Maybe @obenland would know?

Reply:
I'm tempted to remove this _dual_ behavior. The general idea is being treated in this https://github.com/WordPress/gutenberg/pull/17930.

Reply:
Yeah, probably worth keeping the way it was and dealing with it separately.